tests/test_kernel_module.py

```diff
@@ -48,6 +48,34 @@ def create_kernel_test(containerfile: str) -> ParameterSet:
 )
 
 
+# the commit hash and the download link for the sources need to be updated together
+# there is no significance to the chosen hash, it is simple the tip of tree with the
+# corresponding sources to fetch.
+KMP_CONTAINER = create_kernel_test(
+    r"""WORKDIR /src
+RUN set -euxo pipefail; \
+    curl -sf https://src.opensuse.org/pool/vhba-kmp/archive/f7b947967abfd854991eed3a50a333f2efa4c47932e3d7d2fc1d07c6afdb79bd.tar.gz | tar -xzf - ; \
+    cp -a vhba-kmp/* /usr/src/packages/SOURCES; cd /usr/src/packages/SOURCES ; \
+    curl -LO https://downloads.sf.net/cdemu/vhba-module-20240202.tar.xz ; \
+    rpmbuild -bb vhba-kmp.spec ; \
```

Comment:
Fails on SLES 16 with:
```
2.118 + rpmbuild -bb vhba-kmp.spec
2.242 error: Failed build dependencies:
2.242 	suse-kernel-rpm-scriptlets is needed by vhba-kmp-20240202-0.x86_64
```

Reply:
@dcermak yes, thats why there is a  PR fixing it for 3 weeks waiting for review here: https://github.com/SUSE/BCI-dockerfile-generator/pull/2017